bigtable/admin.go

```diff
@@ -1206,15 +1300,39 @@ func (iac *InstanceAdminClient) GetCluster(ctx context.Context, instanceID, clus
 	}
 	nameParts := strings.Split(c.Name, "/")
 	locParts := strings.Split(c.Location, "/")
-	cis := &ClusterInfo{
+	ci := &ClusterInfo{
 		Name:        nameParts[len(nameParts)-1],
 		Zone:        locParts[len(locParts)-1],
 		ServeNodes:  int(c.ServeNodes),
 		State:       c.State.String(),
 		StorageType: storageTypeFromProto(c.DefaultStorageType),
 		KMSKeyName:  kmsKeyName,
 	}
-	return cis, nil
+	// Use type assertion to handle protobuf oneof type
+	if cfg, ok := c.Config.(*btapb.Cluster_ClusterConfig_); ok {
+		if asc, ok := autoscalingConfig(cfg); ok {
+			ci.AutoscalingConfig = asc
+		}
+	}
+	return ci, nil
+}
+
+func autoscalingConfig(c *btapb.Cluster_ClusterConfig_) (AutoscalingConfig, bool) {
```

Comment:
Is there a naming convention for these veneer type to/from proto type funcs? In storage/apiary we say to/from raw usually.

Reply:
to/from seems like the way to go. I'll update here.

Reply:
I think the convention is `toProto()` and `fromProto()`, but I can't swear to it.

Reply:
Updated to fromClusterConfigProto (since fromProto would be ambiguous).